packages/reactive-element/.gitignore

```diff
@@ -1,9 +1,11 @@
 /decorators/
 /development/
 /node/
+/std-decorators/
 /test/
 /css-tag.*
 /decorators.*
+/std-decorators.*
 /polyfill-support.*
```

Comment:
I think we should also publish a /legacy-decorators.js file that just re-exports decorators

Reply:
Thanks for doing this!